src/CreateLesson/CreateLesson.js

```diff
@@ -10,7 +10,6 @@ const CreateLesson = () => {
   const deleteQuestion = (e) => {
     //eslint-disable-next-line
```

Comment:
we could likely use the + to make that a number and not use those disable lines here.